docs/user/accounts.rst

```diff
@@ -126,10 +126,11 @@ Appearance settings allow you to customize the look and feel of the wiki.
 Theme name
  The bundled MoinMoin wiki theme which you would like to use.
 
-User CSS URL
- If you want to style MoinMoin with custom Cascading Style Sheets (CSS), enter a URL for your
+CSS file
+ If you want to style MoinMoin with custom Cascading Style Sheets (CSS), enter a filename for your
  custom stylesheet here. Custom CSS provides an advanced level of control over appearance of
- MoinMoin pages.
+ MoinMoin pages. The file needs to be in the “wiki_local” subdirectory of the wiki instance.
```

Comment:
> “wiki_local”

Out of interest: the use of opening/closing double quotes here is probably legit, but was it intended?